ddsp/training/train_util_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The DDSP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for ddsp.training.test_util."""
+
+import json
+import os
+import tensorflow.compat.v2 as tf
+from tensorflow.python.eager import context
+import train_util
+
+
+class GetStrategyTest(tf.test.TestCase):
+
+  def test_multiworker_strategy_is_set(self):
+    """Check if proper strategy is returned and
+    that cluster configuration for multi worker strategy is set."""
+    # Context reset needed after last test
+    context._reset_context() # pylint: disable=protected-access
+
+    strategy = train_util.get_strategy(cluster_config=
+                                       '{"cluster": \
```

Comment:
In general you should avoid `\` for line continuations, strings can naturally continue from line to line if they are inside a (),

Ex.
('This is a totally valid '
 'way to write a long string.')

Try to prefer long lines that use the 80 spaces alloted to you. For example, you can put cluster_config on a new line with 4 space indent to give everything else more space

Reply:
Done. Hope this formatting looks better.

Reply:
Looks good